test/itest_reconnect.py

```diff
@@ -0,0 +1,48 @@
+
+import testlib
+
+def simple(env):
+    print('Starting nodes')
+    lit1 = env.new_lit_node()
+    lit2 = env.new_lit_node()
+    print('Connecting nodes...')
+    lit1.connect_to_peer(lit2)
+    l2p1 = lit1.get_peer_id(lit2)
+    print('OK, shutting down node 1...')
+    lit1.shutdown()
+    print('Restarting...')
+    lit1.start()
+    print('Connecting to node 2...')
+    lit1.connect_to_peer(lit2)
+    l2p2 = lit1.get_peer_id(lit2)
+    print('Checking IDs match...')
+    print('Node 2: %s -> %s' % (l2p2, l2p2))
+    assert l2p1 == l2p2, 'peer IDs on node 1 don\'t match across restarts!'
+    print('OK')
+
+def reordered(env):
+    print('Starting nodes')
+    lit1 = env.new_lit_node()
+    lit2 = env.new_lit_node()
+    lit3 = env.new_lit_node()
+    print('Connecting nodes... (2 then 3)')
+    lit1.connect_to_peer(lit2)
+    lit1.connect_to_peer(lit3)
+    l2p1 = lit1.get_peer_id(lit2)
```

Comment:
Again, the test looks good, but we could do with better names since it would be easier for people to understand this in the future.

Reply:
yeah I guess I could comment that